libs/template/templates/default-python/template/{{.project_name}}/resources/{{.project_name}}_job.yml.tmpl

```diff
@@ -10,20 +9,41 @@ resources:
         timezone_id: Europe/Amsterdam
 
       {{- if not is_service_principal}}
+
       email_notifications:
         on_failure:
           - {{user_name}}
+
+      {{else}}
+
       {{end -}}
 
       tasks:
+        {{- if eq .include_notebook "yes" }}
         - task_key: notebook_task
           job_cluster_key: job_cluster
           notebook_task:
             notebook_path: ../src/notebook.ipynb
-
-        - task_key: python_wheel_task
+        {{end -}}
+        {{- if (eq .include_dlt "yes") }}
+        - task_key: refresh_pipeline
+          {{- if (eq .include_notebook "yes" )}}
           depends_on:
             - task_key: notebook_task
+          {{- end}}
+          pipeline_task:
+            {{- /* FIXME: this should pipeline_id: ./{{project_name}}_pipeline.yml */}}
```

Comment:
We will need some solution here that doesn't use magics. Or we need hacks in the UI code. And this wouldn't require us to be strict about 1 resource 1 file. But it would only work if there is only one pipeline in that file.